spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt

```diff
@@ -26,7 +26,7 @@ import com.fasterxml.jackson.annotation.JsonValue
  *  References an external source of additional information, metadata, enumerations, asset identifiers, or downloadable
  *  content believed to be relevant to the Package.
  */
-class SpdxExternalReference(
+data class SpdxExternalReference(
```

Comment:
Commit message: For consistency with what? I assume you mean with the other SPDX classes, so please say "For consistency with the other SPDX classes in the model package."

Reply:
Done.